brion/blueConfig.h

```diff
@@ -92,6 +92,32 @@ class BlueConfig : public boost::noncopyable
     }
     //@}
 
+    /** @name Semantic read API */
+    //@{
+    /** @return the full path to the circuit.mvd2 file. @sa Circuit */
```

Comment:
That would be the case in general for all returned sources. spike and report source are URIs, but synapse and circuit sources are strings btw, should be the same for all.